midcli/__main__.py

```diff
@@ -244,9 +244,11 @@ def run(self):
 
 def main():
     parser = argparse.ArgumentParser()
-    parser.add_argument('--websocket')
+    parser.add_argument('--url')
     parser.add_argument('--user')
     parser.add_argument('--password')
+    parser.add_argument('--timeout', type=int, default=1200,
+                        help='Timeout for executing non-job commands')
```

Comment:
Just a nit but would probably be beneficial to add `(in seconds)` to the `help=` text